docs/content/documentation/systems/Postprocessors/framework/AxisymmetricCenterlineAverageValue.md

```diff
@@ -1,8 +1,12 @@
-<!-- MOOSE Documentation Stub: Remove this when content is added. -->
-
 # AxisymmetricCenterlineAverageValue
 !syntax description /Postprocessors/AxisymmetricCenterlineAverageValue
 
+## Description
+'AxisymmetricCenterlineAverageValue' is a specialized case of the [SideAverageValue](/SideAverageValue.md) postprocessor for cases in which the sideset on which the average value is being calculated is along the centerline axis in axisymmetric simulations.
```

Comment:
I think this was your code contribution.  Anything more you'd like to see in the description?